src/components/Modals/GeneralRoomModal.jsx

```diff
@@ -61,7 +61,25 @@ function ModalStepIcon(props) {
 }
 
 const SuccessPanel = ({ createdOrEditedRoom, handleCloseModal }) => {
-  const gameUrl = getGameUrl(createdOrEditedRoom.code);
+  const { setSnackbar } = useSnackbar();
+  const roomCode = createdOrEditedRoom.code;
+  const gameUrl = getGameUrl(roomCode);
+
+  const copyGameUrl = () => {
+    // Note: this clipboard functionality may not be supported on some older browsers
+    navigator.clipboard.writeText(gameUrl);
+    setSnackbar({
+      message: 'Successfully copied game URL!',
+      open: true,
+      type: 'success',
+    });
+  };
+
+  const headerText = `ToBeYou.sg: Invitation to join room ${roomCode}`;
+  const bodyText = `You've been invited to join the To Be You game room ${roomCode}. Go to ${gameUrl} to get started!`;
+
+  const mailText = `mailto:email@example.com?subject=${headerText}&body=${bodyText}`;
+  const whatsappText = `https://api.whatsapp.com/send?text=${bodyText}`;
```

Comment:
Are these messages ok for sharing via mail or whatsapp? @gauravkeerthi

Reply:
ok